src/Core/src/Platform/Android/MauiAppCompatActivity.cs

```diff
@@ -57,18 +53,26 @@ protected override void OnCreate(Bundle? savedInstanceState)
 				window = mauiApp.CreateWindow(state);
 			}
 
+			AlertManager.Subscribe(this, mauiContext);
```

Comment:
Can we make the AlertManager more integrated into the MAUI bits instead of being on the individual activity/applications ? 

They all seem to share the same "subscribe" API that can take a MauiContext and then from there they can process that Context

And then much like the FontManager  where we retrieve the required service via the ServiceProvider